pom.xml

```diff
@@ -437,10 +421,37 @@
           </reportSet>
         </reportSets>
       </plugin>
-     </plugins>
+    </plugins>
   </reporting>
 
   <profiles>
+    <!-- JaCoCo for test coverage -->
```

Comment:
@hohwille Was this how you meant to include the `jacoco-report` profile into maven-parent? I thought this was a good way of doing it. After this PR is merged, we can also delete the profile from the pom.xml of sonar-devon4j-plugin.